### PR TITLE
[Ready to review] README Spanish translation

### LINKED ISSUE #1337: [Translate] README Spanish translation
### Summary

As suggested by @malvikasharan in #1335, @camachoreina and I we will be translating the README to Spanish.
### What needs to be done?

- [ ] *README.md translation*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by camachoreina**
### Summary

Fixes #1337 

### List of changes proposed in this PR (pull-request)

* Add a new README.md translated to Spanish: README-Spanish.md

### What should a reviewer concentrate their feedback on?

Reviewer should be Spanish-speaker.

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: 

@all-contributors please add @camachoreina and @giannidsp for translation

### CODE REVIEW

**Comment by malvikasharan** [comment]
Hi @ChristianECG, I hope you saw that Spanish translation is already happening. You can help review that PR once its ready.

**Comment by crangelsmith** [comment]
We plan to merge the work @giannidsp from https://github.com/giannidsp/the-turing-way/pull/1/commits to the already translated version.

File: README-Spanish.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+_The Turing Way_ es una guía ligeramente testaruda para la ciencia de datos reproducible.
```
Comment (lauracion) [inline README-Spanish.md:9]: Sugiero "asertiva" en lugar de "testaruda". Me parece que testaruda tiene una connotación ligeramente más negativa que asertiva.

File: README-Spanish.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+_The Turing Way_ es una guía ligeramente testaruda para la ciencia de datos reproducible. 
+Puedes leerla aquí:: <https://the-turing-way.netlify.com>
+Actualmente estás en el repositorio de GitHub del proyecto donde se encuentran todos los fragmentos que componen la guía y donde ocurre el proceso de escritura / construcción de la misma.
```
Comment (lauracion) [inline README-Spanish.md:11]: Sugiero "escritura/construcción" en lugar de "escritura / construcción"

File: README-Spanish.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Nuestro objetivo es proporcionar toda la información que los investigadores necesitan al inicio de sus proyectos para garantizar que sean fáciles de reproducir al final.
```
Comment (lauracion) [inline README-Spanish.md:13]: Sugiero el uso de lenguaje neutro. En lugar de "los investigadores" podría ser "las personas que hacen investigación" o "las y los investigadores", aunque ésta última forma excluye a las personas no binarias.

File: README-Spanish.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Esto también significa asegurarse de que los estudiantes de doctorado, los post-doctorantes, los investigadores principales y los equipos de financiación sepan qué partes de la "responsabilidad de reproducibilidad" poseen y qué deben hacer para impulsar que la ciencia de datos sean más eficiente, efectiva y comprensible.
```
Comment (lauracion) [inline README-Spanish.md:15]: Sugiero cambiar "asegurarse de que los estudiantes de doctorado, los post-doctorantes, los investigadores principales" por "asegurarse de que estudiantes de doctorado, post-doctorantes, investigadoras e investigadores principales"

File: README-Spanish.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+Tabla de contenidos:
+
+- [Acerca el proyecto](#about-the-project)
```
Comment (lauracion) [inline README-Spanish.md:19]: "Acerca el proyecto" tal vez debería ser "Acerca del proyecto"

File: README-Spanish.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+- [Contáctanos](#get-in-touch)
+- [Colaboradores](#contributors)
+
+🎧 Si prefieres una introducción de audio al proyecto (en inglés) Rachel, una colaboradora del proyecto presentó en el [Open Science Fair 2019](https://www.opensciencefair.eu/) en Oporto y su presentación fue grabada por el [Podcast Orion](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
+la descripción general del _The Turing Way_ comienza en el minuto 5:13. 
+
+### Acerca del proyecto
+
+La investigación reproducible es necesaria para garantizar que se pueda confiar en el trabajo científico.
+Los financiadores y editores científicos están comenzando a exigir que las publicaciones incluyan acceso a los datos subyacentes y al código de análisis.
```
Comment (lauracion) [inline README-Spanish.md:32]: Sugiero cambiar "Los financiadores y editores científicos" por "Quienes financian y editan trabajos científicos"

File: README-Spanish.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+El objetivo es garantizar que todos los resultados puedan verificarse de forma independiente y servir como base para trabajos futuros.
+Esto a veces es más fácil decirlo que hacerlo.
+Compartir estos resultados de investigación significa comprender la gestión de datos, las bibliotecas, el desarrollo de software y las técnicas de integración continua: habilidades que no se enseñan ni se esperan de los investigadores académicos y científicos de datos. Como estas actividades no se enseñan comúnmente, reconocemos que la carga de los requisitos y la adquisición de nuevas habilidades puede ser intimidante para las personas que son nuevas en este mundo.
```
Comment (lauracion) [inline README-Spanish.md:35]: Sugiero cambiar "las bibliotecas" por "bibliotecología" que se corresponde mejor con el original "library sciences"

File: README-Spanish.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+*The Turing Way* es un manual para ayudar a los estudiantes, sus supervisores, financiadores y editores de revistas a garantizar que la ciencia de datos reproducible sea "demasiado fácil como para no hacerla" incluso para personas que nunca antes han trabajado de esta manera.
```
Comment (lauracion) [inline README-Spanish.md:36]: Sugiero cambiar "a los estudiantes, sus supervisores, financiadores y editores de revistas" por "a personas que estudian, supervisan, financian y editan trabajos científicos"

File: README-Spanish.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+Incluirá material de capacitación sobre control de versiones, *analysis testing* y comunicación abierta y transparente con futuros usuarios, y se basará en los estudios de caso y talleres del Instituto Turing.
```
Comment (lauracion) [inline README-Spanish.md:37]: Creo que "analysis testing" puede traducirse como "testeo de análisis". Sugiero cambiar "futuros usuarios" por "futuras personas usuarias".

File: README-Spanish.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+Este es (parte de) el equipo de planificación del proyecto en el Instituto Turing.
+Para obtener más información sobre cómo contactarnos, consulta el [documento de formas de trabajo] (ways_of_working.md).
+
+![Foto del equipo](book/website/figures/TuringWayTeam.jpg)
+
+
+### Contributing
+
+:construction: Este repositorio es siempre un trabajo en progreso y **todos** están invitados a ayudarnos a construir algo que sea útil para muchos. :construction:
```
Comment (lauracion) [inline README-Spanish.md:50]: Sugiero cambiar "**todos** están invitados a ayudarnos a construir algo que sea útil para muchos" por "**todas las personas** están invitadas a ayudarnos a construir algo que sea útil para muchas personas"

File: README-Spanish.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+Se les solicita a quien colaboren con el proyecto que sigan nuestro [código de conducta] (CODE_OF_CONDUCT.md) y que revisen nuestras [pautas de contribución] (CONTRIBUTING.md) para obtener más información sobre cómo comenzar.
```
Comment (lauracion) [inline README-Spanish.md:52]: Sugiero cambiar "Se les solicita a quien colaboren" por "Se le socilita a quienes colaboren"

File: README-Spanish.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+Si no conoces o tienes dudas de contribuir en GitHub, puedes también contribuir con un estudio de caso y enviarnos consejos a través de nuestro [formulario de envío de Google] (https://goo.gl/forms/akFqZEIy2kxAjfZW2).
```
Comment (lauracion) [inline README-Spanish.md:54]: Sugiero cambiar "tienes dudas de contribuir" a "tienes dudas sobre cómo contribuir"

File: README-Spanish.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+Las mismas representan una variedad de contenidos del manual, de los esfuerzos de colaboración en la comunidad y del proyecto _The Turing Way_ en general.
+Estas ilustraciones están disponibles en Zenodo ([https://zenodo.org/record/3695300)](https://zenodo.org/record/3695300) bajo una licencia CC-BY.
+
+Cuando utilises cualquiera de las imágenes, por favor incluye la siguiente atribución:
+> Esta imagen fue creada por Scriberia para la comunidad The Turing Way y se usa bajo una licencia CC-BY.
+
+La última versión de Zenodo se puede citar como:
+> The Turing Way Community, & Scriberia. (2020, March 3). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3695300
+
+Hemos utilizado algunas de estas ilustraciones en las respuestas a las contribuciones de nuevos miembros en el [Bot de bienvenida](https://github.com/apps/welcome) de este repositorio de GitHub.
```
Comment (lauracion) [inline README-Spanish.md:86]: Sugiero cambiar "nuevos miembros" por "personas que se integran a esta comunidad"

File: README-Spanish.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+
+### Contáctanos
+
+Tenemos una sala de chat de gitter y nos encantaría que pasaras a saludar a <https://gitter.im/alan-turing-institute/the-turing-way>.
+Este foro también está sincronizado con [Matrix](https://matrix.org) en [#the-turing-way:matrix.org](https://riot.im/app/#/room/#the-turing-way:matrix.org) y puedes unirte a nosotros allí si lo prefieres.
+
+También tenemos una lista de correos a las que enviamos actualizaciones mensuales del proyecto. Suscríbete en <https://tinyletter.com/TuringWay>.
```
Comment (lauracion) [inline README-Spanish.md:93]: "a las que enviamos" -> "a la que enviamos"

File: README-Spanish.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+
+Puedes ponerte en contacto con nuestro administrador de la comunidad **Malvika Sharan** por correo electrónico a [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk).
```
Comment (lauracion) [inline README-Spanish.md:95]: "nuestro administrador" deberia ser "nuestra administradora"

**Review by lauracion — CHANGES_REQUESTED** [review]
Hi, 
The translation is very well done! Thank you! I left a few suggestions for changes mostly related to the use of neutral-gender Spanish that is more welcoming for persons of any gender.
I hope this helps.
Best,
Laura

**Comment by malvikasharan** [comment]
Wonderful! Thank you @lauracion. @camachoreina and @crangelsmith in addition to Laura's review, can I ask you to do the following as well.

- [ ] update your local branch with recent updates from the main branch
- [ ] move your translated file to the folder `README-translated`
- [ ] remove the test-status table. Also, remove contributor's table detail, and instead add a link to the original table (the reason is that we will update this table constantly and would like to avoid duplication)
  - see this file [for example](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Portuguese.md)
- [ ] Add a link to the README-Spanish on the top of Repository's README, for examples "This README.md file in also available in Dutch ([README-Dutch](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Dutch.md)), Portuguese ([README-Portuguese](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Portuguese.md)) and  Spanish ([README-Spanish](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Spanish.md))"

**Comment by malvikasharan** [comment]
@all-contributors please add @lauracion for translation.

**Comment by camachoreina** [comment]
Thanks a lot @lauracion for the careful review! Indeed, it is very important to make the test neutral-gender. We had some doubts with some of the sentences and we are glad you helped us resolving those!
All your suggestions were implemented. 
Best,
Reina and Camila

**Comment by camachoreina** [comment]
Thanks @malvikasharan! All your requests have been implemented. Please take a look and let us know if anything else is missing. Best!

> Wonderful! Thank you @lauracion. @camachoreina and @crangelsmith in addition to Laura's review, can I ask you to do the following as well.
> 
> * [X ]  update your local branch with recent updates from the main branch
> * [X]  move your translated file to the folder `README-translated`
> * [X ]  remove the test-status table. Also, remove contributor's table detail, and instead add a link to the original table (the reason is that we will update this table constantly and would like to avoid duplication)
>   
>   * see this file [for example](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Portuguese.md)
> * [X]  Add a link to the README-Spanish on the top of Repository's README, for examples "This README.md file in also available in Dutch ([README-Dutch](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Dutch.md)), Portuguese ([README-Portuguese](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Portuguese.md)) and  Spanish ([README-Spanish](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Spanish.md))"

**Comment by malvikasharan** [comment]
Hey @crangelsmith and @camachoreina. the file looks good. So exciting to get this translation added to the project. Muchas gracias <3 please merge it when you are ready!

**Review by malvikasharan — APPROVED** [review]

**Pull request merged**